soc/intel/common/soc_gpio.c

```diff
@@ -39,61 +61,88 @@ static int gpio_info_acpi_get(uint32_t bank_idx, uint32_t field_idx, uint32_t *r
 	return status;
 }
 
-int soc_acpi_gpio_resource_get(int bank_idx, char *hid, char *uid, struct gpio_acpi_res *res)
+int soc_acpi_gpio_resource_get(int bank_idx, char *hid, char *uid, struct gpio_acpi_res *res,
+			       bool ginf)
 {
 	int ret;
 	struct acpi_dev *acpi_child;
 	struct acpi_irq_resource irq_res;
 	struct acpi_mmio_resource mmio_res;
-	uint32_t field_val[7];
+	uint32_t field_val[8] = {0};
 	uint16_t irqs[CONFIG_ACPI_IRQ_VECTOR_MAX];
 	struct acpi_reg_base reg_base[CONFIG_ACPI_MMIO_ENTRIES_MAX];
+	uint8_t uid_int = 0;
 
 	acpi_child = acpi_device_get(hid, uid);
 	if (!acpi_child) {
-		printk("acpi_device_get failed\n");
+		LOG_ERR("acpi_device_get failed");
```

Comment:
Nitpick: maybe have these `s/printk/LOG_ERR` on a different patch, so it doesn't distract from the functional changes?

Reply:
Shall I add this change in a different comment in this PR?